docs/README.md

```diff
@@ -463,11 +463,11 @@ shows something like:
 +-------------------------------------+-------------------+
 | pgroll                              | postgres          |
 | public                              | pg_database_owner |
-| public_02_user_description_not_null | postgres          |
+| public_02_user_description_set_nullable | postgres          |
```

Comment:
same comment about the preceding lines in the table.

Reply:
reformatted